### DISCUSSION TITLE
Woah! 200 stars!

### DISCUSSION THREAD

**Discussion by mbround18** [mbround18/valheim-docker#113] [Show and tell]
Sæll Vikings!

When I started to write this project I originally just wanted to build a very simple server managing tool that would allow me and my friends to run a Valheim server merely to roam around together. I had no idea the repository would blow up like this and have such a great community here in the discussions <3 

Development is a hard processes and can take time especially on side projects like this but you all have made it easy by providing quick feedback, ideas, suggestions, and helping here in the discussions. It is because of your contributions that this project has turned from something simple into a robust project with awesome features.

I wanted to say thank you for being awesome guys <3 it is not every day a project gets to have such an active community to really mold it <3

**Comment by Flightkick** [comment]
Well, thank you for putting in the effort to package the game to a Docker image! Bonus points for best practices like configurable uid/gid and stuff.

Very easy to use! I'll keep an eye out to check if the issues with auto updating in combination with save file loading and the issue with backup sizes have been fixed. Cheers!

**Reply by mbround18** [reply]
The backup issue should hopefully be resolved, it looks like those issues were user error based on where they had volume mounts. They were backing up the whole game into those backup files because the mounted the server installation directory with the save directory in it.

**Reply by TheDakk** [reply]
What's the issue with auto update?

**Reply by Flightkick** [reply]
@TheDakk I was referring to this one https://github.com/mbround18/valheim-docker/discussions/85 but that might also be a user error ;)

EDIT: I can confirm that the server works as expected with both the backup and auto-update features enabled.

**Reply by TheDakk** [reply]
I let it run overnight last night, no issues here as well.

**Comment by neoKushan** [comment]
You've done a great job with this server image, well done! And your interactions and support have been great to see, kudos to you.

**Reply by TheDakk** [reply]
I agree, @mbround18 would love to donate if available.

**Reply by mbround18** [reply]
> I agree, @mbround18 would love to donate if available.

Donations are welcomed but not needed <3 However, if you would like I have included my sponsor information in the repo <3 there should be a sponsor button at the top with all the links.
